graphsdk/src/androidTest/java/com/microsoft/graph/serializer/CalendarSerializerTests.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.graph.serializer;
+
+import android.test.AndroidTestCase;
+
+import java.util.Calendar;
+
+
+/**
+ * Test cases for {@see CalendarSerializer}
+ */
+public class CalendarSerializerTests extends AndroidTestCase {
+
+    public void testCalendarSerializer() throws Exception {
```

Comment:
Still waiting on this change

---
In reply to: [66119339](https://github.com/microsoftgraph/msgraph-sdk-android/pull/17#discussion_r66119339) [](ancestors = 66119339,65755264)